client/src/components/Pages/Help/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -19,6 +19,17 @@ exports[`components/Pages/<Help /> should render component with no feedbacks 1`]
   >
     Notice, that these questions are only examples and you do not have to ask exactly the same questions. My bot should understand many different ways of asking about these topics. So you are welcome try different variations to see how flexible it is.
   </p>
+  <p>
+    If you are lazy and you do not want to play with the bot - here you can find a link to my 
```

Comment:
Not sure about this one, but as it was before and partially it makes sense, I decided to leave it here... :)